tests/integration/test_fim/conftest.py

```diff
@@ -43,6 +43,39 @@ def create_key(request):
                        error_message='Did not receive expected "integrity clear" event')
 
 
+@pytest.fixture(scope='function')
```

Comment:
```suggestion
@pytest.fixture()
```

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/3856/commits/b50ad9625566811faf8e88847b0fb2542f4bcb69